code/modules/mob/living/carbon/human/human.dm

```diff
@@ -949,10 +949,24 @@
 	return (ishuman(target) && !(target.mobility_flags & MOBILITY_STAND))
 
 /mob/living/carbon/human/proc/fireman_carry(mob/living/carbon/target)
+	var/carryspeed = 0 //if you have latex you are faster at grabbing

```

Comment:
default to 50 so you don't need the else

Reply:
This is turbo nitpicky but wouldn't `carrydelay` be a more accurate representation of what the var is? Weird var names trigger me.

Reply:
did these